Library/Homebrew/test/formula_installer_bottle_spec.rb

```diff
@@ -5,7 +5,14 @@
 require "test/support/fixtures/testball"
 require "test/support/fixtures/testball_bottle"
 
-describe FormulaInstaller do
+describe FormulaInstaller, :integration_test do
```

Comment:
Are you sure the `:integration_test` bit is required?

Reply:
To be more explicit: this definitely shouldn't be an integration test as that'll shell out to a new `brew` process unnecessarily.

Reply:
I used `:integration_test` to gain access to `setup_test_formula`. I'll try using `stub_formula_loader formula("patchelf")` instead.

Reply:
I've fixed this issue. This test no longer requires `:integration_test`.